nanoFramework build is broken

@josesimoes do you have any insight on why this broke recently? It seems the VS extensions had an update Dec 21.

.NET nanoFramework fails to build, with no change in git since last successful build.

Last successful build Dec 7, 2021:
https://ci.appveyor.com/project/angularsen/unitsnet/builds/41801911

Failed build Dec 28, 2021:
https://ci.appveyor.com/project/angularsen/unitsnet/builds/42022265#L4072

[log.txt](https://github.com/angularsen/UnitsNet/files/7785679/log.txt)

> C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\nanoFramework\v1.0\NFProjectSystem.MDP.targets(740,5): error MSB4062: The "GenerateBinaryOutputTask" task could not be loaded from the assembly C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\nanoFramework\v1.0\nanoFramework.Tools.BuildTasks.dll. Could not load file or assembly 'file:///C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\nanoFramework\v1.0\nanoFramework.Tools.BuildTasks.dll' or one of its dependencies. An attempt was made to load a program with an incorrect format. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask. [C:\projects\unitsnet\UnitsNet.NanoFramework\GeneratedCode\Acceleration\Acceleration.nfproj]


The `init.ps1` script seems to do its thing:
```
[00:00:22] Initializing...
[00:00:22] Download dotnet-reportgenerator-globaltool...
[00:00:26] You can invoke the tool using the following command: reportgenerator
[00:00:26] Tool 'dotnet-reportgenerator-globaltool' (version '5.0.0') was successfully installed.
[00:00:26] Download dotnet-reportgenerator-globaltool...OK.
[00:00:26] Installing .NET nanoFramework VS extension...
[00:00:34] Installed VS extension v2019.10.0.6
[00:00:34] Initialized.
```

I see from the [vsix gallery feed](https://www.vsixgallery.com/feed/author/nanoframework) that there are both VS2019 and VS2022 extensions listed, but it did pick the 2019 one.
AppVeyor is configured with VS2019 image.

I have a suspicion that the recent update to VS extensions is related, but can't immediately see what is wrong.

--- a/Build/init.ps1
+++ b/Build/init.ps1
@@ -29,9 +29,17 @@ if (!(Test-Path "$msbuildPath/nanoFramework")) {
   $webClient.DownloadFile("http://vsixgallery.com/feed/author/nanoframework", $vsixFeedXml)
   [xml]$feedDetails = Get-Content $vsixFeedXml
 
-  $extensionUrl = $feedDetails.feed.entry[1].content.src
+  foreach ($entry in $feedDetails.feed.entry)
+  {
+      if($entry.id -eq '455f2be5-bb07-451e-b351-a9faf3018dc9')
+      {
+          $extensionUrl = $entry.content.src
+          $extensionVersion = $entry.Vsix.Version
+          break
+      }
+  }
+
   $vsixPath = Join-Path -Path $tempDir -ChildPath "nf-extension.zip"
-  $extensionVersion = $feedDetails.feed.entry[0].Vsix.Version
   $webClient.DownloadFile($extensionUrl,$vsixPath)
   Expand-Archive -LiteralPath $vsixPath -DestinationPath $tempDir\nf-extension\ | Write-Host
 

